config/environments/production.rb

```diff
@@ -22,23 +22,21 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
 
-  # Compress JavaScripts and CSS.
+  # Compress CSS using a preprocessor.
```

Comment:
This comment is quite confusing as we've commented out the CSS preprocessor on line 27. 🤔

Reply:
yeah, fair play, this happened automatically when I upgraded. Will change it back :)